.github/workflows/check-formatting.yml

```diff
@@ -19,20 +19,21 @@ jobs:
       - name: Setup Go environment
         uses: actions/setup-go@v3
         with:
-          go-version: '^1.19.2'
+          go-version: 'stable'
 
       - name: Install formatters
         run: |
           wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
           sudo apt-get install software-properties-common
-          sudo add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-13 main'
-          sudo apt-get install clang-format-13
-          curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.42.1/ktlint && chmod a+x ktlint && sudo mv ktlint /usr/bin/ktlint
+          sudo add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-15 main'
+          sudo apt-get install clang-format-15
+          curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.48.0/ktlint && chmod a+x ktlint && sudo mv ktlint /usr/bin/ktlint
           go install github.com/google/addlicense@latest
           go install github.com/bazelbuild/buildtools/buildifier@latest
 
       - name: Run format.sh and print changes
         run: |
+          shopt -s expand_aliases
```

Comment:
Didn't know this, could you explain what it does in this context?

Reply:
This was an attempt to make the `clang-format` alias available when executing the format script. I usually install the LLVM toolchain from the GitHub releases, and the tools are not suffixed with a version number. I've adjusted the format script to use `clang-format` and used LLVM 14 in the GitHub action.